app/src/main/java/org/mifos/mobile/ui/fragments/TransferProcessFragment.java

```diff
@@ -193,4 +192,21 @@ public void onDestroyView() {
         super.onDestroyView();
         presenter.detachView();
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        switch (requestCode) {
```

Comment:
Why switch is used when there is single case?

Reply:
Actually I did this if in case this method is used in future for another use. 
Should I remove it ?

Reply:
We can always put switch in future if need arises. Please remove it as of now.

Reply:
@luckyman20  I have updated the PR , please review it . Thanks!